exit-status.md

```diff
@@ -0,0 +1,67 @@
+---
+RFC: unassigned
+Author: Nicholas Carpenter <ncarpenter@ebsco.com>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Signal outside tools of specific Chef-Client run status.  Ability to determine results of a Chef-Client run.
+
+## Motivation
+    As a Chef user,
+    I want to be able to determine when a chef-client run is rebooting the node,
+    so that Test-Kitchen/Vagrant/any outside tool can wait for node to reboot, and continue converging.
+    
+## Specification
+* Chef-apply, Chef-client, Chef-Solo should honor the below exit chef run exit codes
+
+### Exit codes reserved across platforms
+* Windows- [Link](https://msdn.microsoft.com/en-us/library/windows/desktop/ms681381(v=vs.85).aspx)
+* Linux - [Sysexits](http://www.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD+4.3-RELEASE&format=html), [Bash Scripting](http://tldp.org/LDP/abs/html/exitcodes.html)
+ 
+
+### Exit Codes usable across platforms
+All exit codes defined should be usable on all supported Chef Platforms.  Also the exit codes used should be idential across platforms.  That limits the total range from 1-255.  Exit codes not explicitly used by Linux/Windows are listed below.  There are a total of 59 exit codes that overlap between the two platforms.
+ 
+ * Exit Codes Available for Chef use:
+     * ~~35,37,40,41,42~~,43,44,45,46,47,48,49,79,81,90,91,92,93,94,95,96,97
```

Comment:
This is to mention that they are strike-through.  In other words these codes are used in the spec.